autopts/bot/iut_config/zephyr.py

```diff
@@ -254,6 +254,7 @@
         "pre_overlay": "overlay-mesh.conf",
         "overlay": {
             'CONFIG_BT_SETTINGS': 'y',
+            'CONFIG_SECURE_STORAGE': 'y',
```

Comment:
We need to add this to every config where we have 'CONFIG_BT_SETTINGS': 'y'

Reply:
update
BTW I wonder if those (and others supported mesh kconfigs) could be simply enabled in mesh overlay for all tests   (and we just disable in overlays here is test explicitly requires feature to be disabled)